jina/serve/stream/helper.py

```diff
@@ -1,18 +1,48 @@
-from typing import Iterator, AsyncIterator, Union
+import asyncio
+from typing import AsyncIterator, Iterator, Optional, Union
 
 from jina.helper import get_or_reuse_loop
 
 
+class RequestsCounter:
```

Comment:
I am wondering if we should not name it `_RequestsCounter`. I remember that in the past we have been locked into not refactoring helper function (rename them or move them) because they were not private and thus users could rely on it. If it is a private Class it kind of gave the right to refactor as we want.

Reply:
I will do later in another PR related to this part of the code

Reply:
sure